### ISSUE TITLE
html-proofer 3.8.0 contains the patch we needed about trailing / - upgrade from custom

### ISSUE THREAD

**Issue by christian-posta** [istio/istio.github.io#849]
https://github.com/istio/istio.github.io/blob/master/Gemfile#L6

@ldemailly ^^

I have an upcoming PR to fix broken links and i've updated to latest html proofer 3.7.0. That should be sufficient, right?

**Comment by ldemailly** [comment]
because my pr to html-proofer wasn’t merged until recently - I am not sure it’s been tagged yet.

please assign reciew to me for your upcoming PR

**Comment by christian-posta** [comment]
okay sounds like we'll wait until next html proofer release with your changes. thanks!

**Comment by ldemailly** [comment]
looks like 3.8.0 has my pr, so you can give it a shot

https://github.com/gjtorikian/html-proofer/commits/master

**Comment by christian-posta** [comment]
ah! cool.. will take a look.

**Comment by christian-posta** [comment]
So unfortunately, after a bit of digging, seems like github-pages dependency has a transitive dependency on `typhoeus` 0.7 (https://github.com/github/pages-health-check/blob/master/github-pages-health-check.gemspec#L20).

This collides with `html-proofer` 3.8.0 which uses `typhoeus` 1.3 (https://github.com/gjtorikian/html-proofer/blob/v3.8.0/html-proofer.gemspec#L23). 

I'm not super familiar with ruby/gemfiles, but after some digging seems like there's no really good way to override/exclude dependencies to make these line up. Seems like lots of gems are using 1.3 of tythoeus but gh-pages is a bit behind. 

Open to suggestions to work around this. 

Changing Gemfile to:

```
source "https://rubygems.org"

gem "github-pages", group: :jekyll_plugins
gem "jekyll-include-cache", "~> 0.1"
gem "nokogiri", ">= 1.8.1"
gem "html-proofer", "3.8.0"
gem "rake"
```

Gives this when `bundle install`:

```
Bundler could not find compatible versions for gem "typhoeus":
  In snapshot (Gemfile.lock):
    typhoeus (= 0.8.0)

  In Gemfile:
    github-pages was resolved to 172, which depends on
      github-pages-health-check (= 1.3.5) was resolved to 1.3.5, which depends on
        typhoeus (~> 0.7)

    html-proofer (= 3.8.0) was resolved to 3.8.0, which depends on
      typhoeus (~> 1.3)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by ldemailly** [comment]
that's odd because I already had that newer typhoeus in the fork:
https://github.com/ldemailly/html-proofer/blob/master/html-proofer.gemspec#L23
and it worked somehow
try `bundle update` ?
